Content.Shared/Dataset/LocalizedDatasetPrototype.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Dataset;
+
+/// <summary>
+/// A variant of <see cref="DatasetPrototype"/> intended to specify a sequence of LocId strings
+/// without having to copy-paste a ton of LocId strings into the YAML.
+/// </summary>
+[Prototype("localizedDataset")]
```

Comment:
the attribute would automatically name this

Reply:
Ah, yes it would. I'll fix that.